Makefile

```diff
@@ -840,16 +840,12 @@ RUNTIME_LL_COMPONENTS = \
   wasm_math \
   win32_math \
   x86 \
+  x86_amx \
   x86_avx \
   x86_avx2 \
   x86_avx512 \
   x86_sse41
 
-ifeq (,$(findstring $(LLVM_VERSION_TIMES_10), 110 111))
-# x86_amx.ll won't compile under LLVM11, but we don't need it there, either
-RUNTIME_LL_COMPONENTS += x86_amx
-endif
-
```

Comment:
You might want to make a similar change here:

https://github.com/halide/Halide/blob/6071cf644b656b43a6de8b5d2d2d364d30678864/src/runtime/CMakeLists.txt#L116-L119

Reply:
This is the only usage of `Halide_LLVM_VERSION`, which is defined here and I am tempted to remove...

https://github.com/halide/Halide/blob/6071cf644b656b43a6de8b5d2d2d364d30678864/dependencies/llvm/CMakeLists.txt#L23-L24

Reply:
> You might want to make a similar change here:
> 
> https://github.com/halide/Halide/blob/6071cf644b656b43a6de8b5d2d2d364d30678864/src/runtime/CMakeLists.txt#L116-L119

done

Reply:
> This is the only usage of `Halide_LLVM_VERSION`, which is defined here and I am tempted to remove...
> 
> https://github.com/halide/Halide/blob/6071cf644b656b43a6de8b5d2d2d364d30678864/dependencies/llvm/CMakeLists.txt#L23-L24

I predict we will eventually need it again. Maybe keep with a comment.